src/work-allocation-2/enums/case-action-ids.ts

```diff
@@ -0,0 +1,10 @@
+export enum CaseActionIds {
+  ASSIGN = 'assign',
```

Comment:
_Question:_
Are all of these valid actions? Some of them look specific to tasks. 🤔

Reply:
This will be addressed in https://tools.hmcts.net/jira/browse/EUI-3600